packages/typespec-ts/test/modularIntegration/customClientRequestId.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { RequestIdClient } from "./generated/headers/client-request-id/src/index.js";
+import { assert } from "chai";
+describe("RequestIdClient Classical Client", () => {
+  let client: RequestIdClient;
+
+  beforeEach(() => {
+    client = new RequestIdClient({
+      allowInsecureConnection: true
+    });
+  });
+
+  it("should add client-request-id in header transparently", async () => {
```

Comment:
Can you add a test to pass the customizedHeaderName in the client constructor ?

Reply:
Added